### PR TITLE
Use only one scenario to test all kafka versions

### PR DESCRIPTION
**Pull request by ryarnyah**
Use only one scenario to avoid code duplicate on molecule tests.

@see https://travis-ci.org/ryarnyah/ansible-kafka-admin/builds/388979874 for build results

### CODE REVIEW

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    group_vars:
+      all:
+        ansible_kafka_supported_versions:
+          - protocol_version: "0.11.0"
+            instance_suffix: "01101"
+          - protocol_version: "1.0.0"
+            instance_suffix: "100"
+          - protocol_version: "1.0.1"
+            instance_suffix: "101"
+          - protocol_version: "1.0.1"
```
Comment (StephenSorriaux) [inline molecule/default/molecule.yml]: I guess it should be 1.1.0 here

**Comment by StephenSorriaux** [comment]
This is awesome! Thank you

File: molecule/default/molecule.yml (hunk 1 above)
Comment (ryarnyah) [reply molecule/default/molecule.yml]: When i refactor this code Cf https://github.com/StephenSorriaux/ansible-kafka-admin/blob/master/molecule/kafka_1.1.0/playbook.yml#L15 i had seen that your are using 1.0.1 for kafka 1.1.0.

Seeing that, i found that kafka-python does not seem to support 1.1.0 API yet, so it seem that 1.0.1 might be a good choice. (Cf https://github.com/dpkp/kafka-python/issues/1512)

Comment (StephenSorriaux) [reply molecule/default/molecule.yml]: You're right, let's keep it that way for now

**Pull request merged**